README.md

```diff
@@ -1 +1,161 @@
-# Ruby-capstone
+<div align="center">
+ <img src="images/logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+  <h1><b>Catalog of My Things</b></h1>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+## 📗 Table of Contents
+
+- [📖 Catalog of My Things ](#-catalog-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+    - [Live Demo](#live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+  - [👥 Authors ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Catalog of My Things <a name="about-project"></a>
+
+**Catalog of My Things** is an application with a library management system; users can interact with and manage a diverse collection of items, including books, music albums, and games, and associate them with labels, genres, and authors. The system also applies rules for archiving items based on criteria specific to each item type.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Data Preservation**
+- **Class implementation with methods**
+- **Add and list the classes**
+- **Unit tests**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!--LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://drive.google.com/file/d/1XbuGeUOACbXScL0wWDzxogPq66pkYbw4/view?usp=sharing)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] Hi, You have done a wonderful work. Can you try to make your Readme look more professional? 
- I would suggest naming this link `Video presentation`, not a live demo link. I am asking this because this video is about the presentation of your wonderful App(which included interesting code)